packages/fury-adapter-oas3-parser/lib/parser/oas/parseServerObject.js

```diff
@@ -0,0 +1,49 @@
+const R = require('ramda');
+const { createWarning } = require('../../elements');
+const {
+  createInvalidMemberWarning, createUnsupportedMemberWarning,
+} = require('../annotations');
+const {
+  isObject, hasKey, isExtension,
+} = require('../../predicates');
+const parseObject = require('../parseObject');
+const parseString = require('../parseString');
+const pipeParseResult = require('../../pipeParseResult');
+
+const name = 'Server Object';
+const requiredKeys = ['url'];
+
+const parseMember = context => R.cond([
+  [hasKey('description'), parseString(context, name, false)],
+  [hasKey('url'), parseString(context, name, true)],
+  // [hasKey('variables'), R.compose(parseServerVariablesObject(context), getValue)], // NOT SUPPORTED YET
+  [hasKey('variables'), createUnsupportedMemberWarning(context.namespace, name)],
+  [isExtension, () => new context.namespace.elements.ParseResult()],
+  [R.T, createInvalidMemberWarning(context.namespace, name)],
+]);
+
+/**
+ * Parse the OpenAPI 'Server Object' (`#/server`)
+ * @see http://spec.openapis.org/oas/v3.0.3#server-object
+ * @returns ParseResult<Link>
```

Comment:
```suggestion
 * @returns ParseResult<Resource>
```

We're returning a resource element with this code.

Reply:
right, changed

Reply:
right, change on next commit